pkg/userutils/set_uid_gid.go

```diff
@@ -1,5 +1,5 @@
-//go:build linux && go1.16
-// +build linux,go1.16
+//go:build linux
```

Comment:
We are no longer supporting go 1.15? Will this break the cw agent?

Reply:
We only build with 1.17+ since we had some breaking dependencies that only worked on 1.17 (build annotations changed). It looks like it has been fixed https://github.com/jaegertracing/jaeger/blob/v1.28.0/cmd/agent/app/servers/thriftudp/socket_buffer.go. I can try to rebuild with 1.15 and see if there are still issues.

The CW agent currently includes ADOT Collector v0.11.0.

Reply:
It doesn't look like we can build with 1.15 even without this.

Reply:
The CW agent dependency is a related dependency but does not block release. Let's separate the two discussions.